### ISSUE TITLE
sql server instance support? host paramter support (local)?

### ISSUE THREAD

**Issue by jiangjianxiao** [cretz/node-tds#2]
host:"(local)"  -> ENOTFOUND
host:"localhost\sqlexpress"  -> ENOTFOUND

**Comment by cretz** [comment]
The host should support "localhost" or "127.0.0.1". This is literally the host used in the socket: https://github.com/cretz/node-tds/blob/master/src/tds-client.coffee#L42. There are some undocumented connections options you can provide such as instanceName which will choose the instance (actually sent as part of a TDS prelogin packet)

I will leave this issue open until I document some of those other connection options.

**Comment by jonbuffington** [comment]
@cretz Is instanceName one of the Connection option keys available in 0.1.0?

**Comment by cretz** [comment]
Yes, it should be, but just not documented. I have not yet tested its functionality.
